plugins/hypervisors/ovm/src/com/cloud/ovm/hypervisor/OvmFencer.java

```diff
@@ -102,6 +103,11 @@ public Boolean fenceOff(VirtualMachine vm, Host host) {
                     s_logger.debug("Moving on to the next host because " + h.toString() + " is unavailable");
                 }
                 continue;
+            } catch (OperationCancelledException e) {
```

Comment:
In case of OperationCancelled why there is a logic to move to another host, it should be terminated immediately?